### ISSUE TITLE
dockerignore file not acknowledged

### ISSUE THREAD

**Issue by lavalamp-** [JoTrdl/grunt-dock#14]
When passing a build command through grunt-dock, and specifying a directory as the dockerfile argument, the .dockerignore file is not taken into consideration (ergo, all files are transferred as docker context). Issuing a standard "docker build ..." command in the specified directory works just fine.

Sample configuration for grunt-dock:

```
images: {
    'IMAGE NAME: {
        dockerfile: '.',
        options: {
            build: {
                t: 'IMAGE TAG'
            }
        }
    }
}
```

So, again, running this ignores any .dockerignore files in the current directory.

**Comment by JoTrdl** [comment]
I am fixing that and will release a new version soon.
Thanks.

**Comment by JoTrdl** [comment]
I just published on npm the version 1.1.2 which supports '.dockerignore' file.
Thanks.
